Themis/Views/Assessment/CorrectionSidebar/GeneralFeedbackCellView.swift

```diff
@@ -8,46 +8,51 @@
 import SwiftUI
 
 struct GeneralFeedbackCellView: View {
-    @ObservedObject var feedbackModel: FeedbackViewModel
+    @EnvironmentObject var assessment: AssessmentViewModel
+    @EnvironmentObject var cvm: CodeEditorViewModel
+
     @State var showAddFeedback = false
 
     var body: some View {
         VStack(alignment: .leading) {
-            HStack {
-                Text("Add General Feedback")
-                    .font(.title3)
-                Spacer()
-                Button {
-                    showAddFeedback = true
-                } label: {
-                    Image(systemName: "plus")
-                }.font(.title3)
-            }.padding()
-
             List {
-                Section(header: Text("General")) {
-                    ForEach(feedbackModel.generalFeedbacks) { feedback in
-                        FeedbackCellView(feedbackModel: feedbackModel, feedbackID: feedback.id)
+                Section(header: HStack {
```

Comment:
Right now general and inline feedbacks are saved in the same section, this will be changed in the future :)